backend/src/index.ts

```diff
@@ -0,0 +1,151 @@
+import express, { Request, Response } from 'express';
+import cors from 'cors';
+import { prisma } from './database'
+import { errorResponseBuilder, generateRandomPassword, isValidUser, parseUserForResponse } from './utils';
+
+const app = express();
+app.use(express.json());
+app.use(cors())
+
+// Create new user
+app.post('/users/new', async (req: Request, res: Response) => {
+  const errorBuilder = errorResponseBuilder(res);
```

Comment:
```suggestion
  const errorBuilder = errorBuilder(res);
```
This is just my suggestion to use the same vocabulary to make the cognitive load as little as possible :)